canvas/views/views.py

```diff
@@ -78,3 +81,25 @@ def events_options_view(request):
     return render(request, 'canvas/course_event_options.html', {
         'events': course.events.all(),
     })
+
+
+def create_event_view(request):
+    if request.method == 'POST':
+        form = CreateEventForm(request.POST)
+        if form.is_valid():
+            try:
+                create_event(name=form.cleaned_data['name'], course=form.cleaned_data['course'],
```

Comment:
Using a ModelForm, here you can just say form.save(), and there no need for create_event function.